python/dgl/graphbolt/minibatch.py

```diff
@@ -54,6 +54,19 @@ class MiniBatch:
       value should be corresponding labels to given 'seed_nodes' or 'node_pairs'.
     """
 
+    seeds: Union[
+        torch.Tensor,
+        Dict[str, torch.Tensor],
+    ] = None
+    """
+    Representation of seed node pairs utilized in link prediction tasks and
+    hyperedge tasks.
+    - If `seeds` is a tensor: It indicates a homogeneous graph where the shape
+      of `seeds` is `N*D` representing N node pairs each with D nodes.
```

Comment:
This is not very accurate, the seeds could be 1D or 2D, and when it is 2D, each row does not have to be a node pair.

Probabaly:
```
- If `seeds` is a tensor: it indicates that the seeds originate from a homogeneous graph. It can be either a 1-dimensional or 2-dimensional tensor:
  - 1-dimensional tensor: Each element directly represents a seed node within the graph.
  - 2-dimensional tensor: Each row designates a seed item, which can encompass various entities such as edges, hyperlinks, or other graph components depending on the specific context.
```